pom.xml

```diff
@@ -1,83 +1,70 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
-	<groupId>com.github.moritzfl</groupId>
-	<artifactId>jtopia</artifactId>
-	<version>0.3</version>
-	<packaging>jar</packaging>
+  <parent>
+    <groupId>com.hubspot</groupId>
+    <artifactId>basepom</artifactId>
+    <version>18.3-SNAPSHOT</version>
+  </parent>
 
-	<name>jtopia</name>
-	<url>http://maven.apache.org</url>
+  <artifactId>jtopia</artifactId>
```

Comment:
I think the way we usually do this is to leave the `groupId` intact, and name the version `1.0-hubspot-SNAPSHOT` (just putting `hubspot` in the version number, whatever that is). Example: https://github.com/HubSpot/modernizer-maven-plugin/blob/ee7d9f88ad370cfe04913322878db8fba34eb129/modernizer-maven-plugin/pom.xml#L12

Reply:
I thought I could use the basepom goodies similar to this [one](https://github.com/HubSpot/simple-slack-api/blob/master/pom.xml). I'll change it back and add dependency management